src/Skia/Avalonia.Skia/GeometryGroupImpl.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using Avalonia.Media;
+using SkiaSharp;
+
+#nullable enable
+
+namespace Avalonia.Skia
+{
+    /// <summary>
+    /// A Skia implementation of a <see cref="Avalonia.Media.GeometryGroup"/>.
+    /// </summary>
+    internal class GeometryGroupImpl : GeometryImpl
+    {
+        public GeometryGroupImpl(FillRule fillRule, IReadOnlyList<Geometry> children)
+        {
+            var path = new SKPath
+            {
+                FillType = fillRule == FillRule.NonZero ? SKPathFillType.Winding : SKPathFillType.EvenOdd,
+            };
+
+            var count = children.Count;
+            
+            for (var i = 0; i < count; ++i)
+            {
+                if (children[i]?.PlatformImpl is GeometryImpl child)
```

Comment:
How do we handle a transform here? As I understand this we just add SKPath elements at the origin.

Reply:
Good catch. I think that will need to be added along with render tests. 👍

Reply:
Added tests for children of `GeometryGroup` and `CombinedGeometry` that have transforms. Seems it already worked fine.

Reply:
This works  because a TransformedGeometryImpl is automatically created when a transform is present. For performance reasons it might still be better to apply a transform directly when geometries are combined/ grouped without having an extra clone of each geometry.
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Skia/Avalonia.Skia/TransformedGeometryImpl.cs#L21

Let's leave it as is and if we feel performance might be bad this can be optimized further.